packages/analytics-js-common/src/v1.1/utils/client_server_name.js

```diff
@@ -74,6 +74,7 @@ const clientToServerNames = {
   TIKTOK_ADS: 'TikTok Ads',
   ACTIVE_CAMPAIGN: 'ActiveCampaign',
   SPRIG: 'Sprig',
+  SPOTIFYPIXEL:'SpotifyPixel',
```

Comment:
Is `SpotifyPixel` the display name?

Reply:
@MoumitaM I did not get you.  In my integrations config PR I have set it like this. While testing E2E also it was working fine
<img width="1440" alt="Screenshot 2024-01-18 at 9 18 24 AM" src="https://github.com/rudderlabs/rudder-sdk-js/assets/60211312/18366d13-ac05-4b96-99a7-82d2ba61afaa">
test report : [link](https://www.notion.so/rudderstacks/Spotify-Pixel-Test-Report-d18a24c74560427e814a13480e1afcde?pvs=4)

Reply:
This mapping is from config name --> display name. 
What you should write is `SPOTIFYPIXEL:'Spotify Pixel'` not `SPOTIFYPIXEL:'SpotifyPixel'`.
Server relies on display name to filter destination.
This is for cloud mode event, nothing to do with device mode testing.